{"name": "test_3", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "E   assert False"}, "description": "Test que prueba otra cosa", "start": 1652976010269, "stop": 1652976010269, "uuid": "259c45a0-0e2a-4993-8458-9f4577c5a2ec", "historyId": "de7f32baed47619a4a1b82e3e0fbc334", "testCaseId": "6c04b0da6552e3cc4d5c748b0b7dc318", "fullName": "tests.test_10_allure.TestReports#test_3", "labels": [{"name": "severity", "value": "minor"}, {"name": "tag", "value": "cashin"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_10_allure"}, {"name": "subSuite", "value": "TestReports"}, {"name": "host", "value": "WKSAR1848L.local"}, {"name": "thread", "value": "5592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_10_allure"}]}